### PR TITLE
additional integration docs

### PR DESCRIPTION
**Pull request by Pezi777**

### CODE REVIEW

File: user/travis-ci-for-private.md [hunk 1]
```diff
@@ -23,21 +23,22 @@ the [blog post](https://blog.travis-ci.com/announcing-pull-request-support/) acc
...
-- Users that can push to a repository on GitHub or Bitbucket can trigger, cancel and restart builds, and change its settings.
-- Users that have admin access to a repository on GitHub or Bitbucket can enable/disable it on Travis CI.
+- Users that can access a repository on GitHub, Bitbucket or GitLab can see the build status and logs on Travis CI.
+- Users that can push to a repository on GitHub, Bitbucket or GitLab can trigger, cancel and restart builds, and change its settings.
+- Users that have admin access to a repository on GitHub, Bitbucket or GitLab can enable/disable it on Travis CI.
 
-To keep the access rights up to date, we sync every user account approximately once every 24 hours with GitHub or Bitbucket. You can use the "Sync account" button on [the profile page](https://travis-ci.com/profile) or `travis sync --com` in the CLI to force a sync.
+To keep the access rights up to date, we sync every user account approximately once every 24 hours with GitHub, Bitbucket or GitLab. You can use the "Sync account" button on [the profile page](https://travis-ci.com/profile) or `travis sync --com` in the CLI to force a sync.
```
Comment (AlonaHlobina) [inline user/travis-ci-for-private.md:33]: @weban I do not think CLI can be used to force a sync with BB or GL?  So we have to delete this part.

File: user/private-dependencies-gl.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+As mentioned a few times, it might make sense to create a dedicated CI user for the following reasons:
+
+- The CI user will only have access to the repositories you want it to have access to.
+- You can limit the access to read access.
+- Less risk when it comes to leaking keys or credentials.
+- The CI user will not leave the organization for non-technical reasons and accidentally break all your builds.
+
+In order to do so, you need to register on GitLab as if you would be signing up for a normal user.
+Registering users cannot be automated, since that would violate the GitLab Terms of Service.
```
Comment (AlonaHlobina) [inline user/private-dependencies-gl.md:196]: @weban, please take a closer look at this section to make sure all the keys and tokens are correct for GL

File: user/gl-oauth-scopes.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Gives the app read access to pull requests and collaborate on them. This scope implies repository, giving read access to the pull request's destination repository.
+
+### email
+Ability to see the user's primary email address. This should make it easier to use GitLab as a login provider to apps or external applications.
+
+### account
+Ability to see all the user's account information. Note that this does not include any ability to mutate any of the data.
+
+### team
+The ability to find out what teams the current user is part of. This is covered by the teams endpoint.
```
Comment (AlonaHlobina) [inline user/gl-oauth-scopes.md:45]: I believe In GitLab, they call this a Group

File: user/travis-ci-for-private.md [hunk 4]
```diff
@@ -23,21 +23,22 @@ the [blog post](https://blog.travis-ci.com/announcing-pull-request-support/) acc
...
 
 ## Who has access to the billing details?
 
 Access rights to the Travis CI [billing page](https://travis-ci.com/account/subscription) can be one of the following:
 
-- **Open (default)**: anyone with admin access to at least one private repository belonging to the organization in GitHub or team in Bitbucket has access to the billing information in Travis CI.
+- **Open (default)**: anyone with admin access to at least one private repository belonging to the organization in GitHub/GitLab or team in Bitbucket has access to the billing information in Travis CI.
```
Comment (AlonaHlobina) [inline user/travis-ci-for-private.md:38]: group in GitLab

File: user/travis-ci-for-private.md [hunk 5]
```diff
@@ -90,3 +91,9 @@ If you're using the Travis CI [command line client](https://github.com/travis-ci
 
 If you want to set the privacy status for a Bitbucket repository, follow the [instructions on
 Bitbucket](https://confluence.atlassian.com/bitbucket/make-a-repo-private-or-public-221449724.html).
+
+
+## How can I set the repository privacy status on GitLab?
+
+If you want to set the privacy status for a GitLab repository, follow the [instructions on
+GitLab](FIXME).
```
Comment (AlonaHlobina) [inline user/travis-ci-for-private.md:99]: https://docs.gitlab.com/ee/user/project/settings/

File: user/travis-ci-for-private.md (hunk 1 above)
Comment (weban) [reply user/travis-ci-for-private.md:33]: Yes CLI is not supported yet

File: legal/security.md [hunk 6]
```diff
@@ -79,6 +79,45 @@ However, to allow us to automatically specify SSH keys, service hook configurati
...
+We only manually access your code when explicitly requested by you and only in explicit consent with you, and only to debug and help solve build issues.
+
+### How does Travis access my Assembla account?
+When you sign up for Travis, we collect an OAuth token from Assembla, which allows us to request data from the Assembla API on your behalf. This OAuth token is stored securely in our database and is protected from unauthorized access.
+
+The token is bound to permissions set on Assembla, so please make sure you've read their documentation on access control and API access permissions.
+
+We use this token in these situations, and under no other circumstances than described below.
+
+- To synchronize the repositories and teams you have access to. We use this information to show you the available repositories on your profile page so you can enable or disable building them on Travis.
```
Comment (weban) [inline legal/security.md:108]: spaces instead of teams

Comment (michal-at-travisci) [reply legal/security.md]: fixed by subsequent commit

**Comment by Montana** [comment]
Not sure if deployed yet, but if not will gladly deploy.

**Pull request merged**